src/components/header/Header.scss

```diff
@@ -1,15 +1,48 @@
 @import "../bootstrap/core";
 
 .header {
+  position: relative;
+  top: 0;
+  left: 0;
+  z-index: $zindex-fixed;
+  display: flex;
+  box-sizing: border-box;
```

Comment:
This is the default value from Bootstrap, isn't it? (Multiple occurrences in this PR)

```
*, *::before, *::after {
    box-sizing: border-box;
}
```

Reply:
I forgot about this. Thank you!